tests/simulate/test_hoomd.py

```diff
@@ -96,13 +97,55 @@ def create_gsd(self):
         relentless.mpi.world.barrier()
         return filename
 
-    def test_initialize_from_file(self):
+    def create_lammps_file(self):
+        file_ = self.directory.file("test.data")
+
+        if relentless.mpi.world.rank_is_root:
+            low = [-5, -5, -5 if self.dim == 3 else -0.1]
+            high = [5, 5, 5 if self.dim == 3 else 0.1]
+            snap = lammpsio.Snapshot(N=5, box=lammpsio.Box(low, high))
+            snap.position[:, :2] = [[-4, -4], [-2, -2], [0, 0], [2, 2], [4, 4]]
+            if self.dim == 3:
+                snap.position[:, 2] = [-4, -2, 0, 2, 4]
+            snap.typeid = [1, 1, 2, 2, 2]
+            snap.mass = [0.3, 0.3, 0.1, 0.1, 0.1]
+            lammpsio.DataFile.create(file_, snap)
+        relentless.mpi.world.barrier()
+
+        return file_
+
+    def test_initialize_from_gsd_file(self):
         ens, pot = self.ens_pot()
-        f = self.create_gsd()
+        f = self.create_gsd_file()
         op = relentless.simulate.InitializeFromFile(filename=f)
         h = relentless.simulate.HOOMD(op)
         h.run(pot, self.directory)
 
+    def test_initialize_from_lammps_file(self):
+        """Test running initialization simulation operations."""
+        if self.dim == 3:
```

Comment:
Same question here about why this is part of the test?

Reply:
Here, we do have to make the potentials because we don't currently support remapping from LAMMPS types to relentless types on initialization. We could reconsider this later.